driver/tests/swe_roe/ex2b_ic_file_with_adapt.yaml

```diff
@@ -0,0 +1,60 @@
+# This test input corresponds to toy-problems/swe_roe/ex2b.c, but uses a file
+# to specify the initial conditions (and a corresponding Exodus mesh).
+
+physics:
+  flow:
+    mode: swe
+
+numerics:
+  spatial: fv
+  temporal: euler
+  riemann: roe
+
+logging:
+  level: debug
+
+time:
+  final_time        : 0.005
+  coupling_interval : 0.001
+  unit              : hours
+  adaptivity:
```

Comment:
What do you think about using `adaptive` instead of `adaptivity` here? It's easier to spell, two-syllables shorter, and has essentially the same meaning. It's technically an adjective instead of a noun, but I think there's a limit to the value of consistency when it comes to practical considerations. :-)